app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -446,7 +446,7 @@ msgid "Year"
 msgstr "Blwyddyn"
 
 #: app/templates/partials/answers/date.html:24
-msgid "YYYY"
+msgid "yyyy"
```

Comment:
This shouldn't have changed, it's the placeholder text for the field.

Reply:
Good spot, have reverted and regenerated the pot, this reference doesn't even exist in the pot file. 👀